pkgs/data/misc/clash-geoip/default.nix

```diff
@@ -25,7 +25,7 @@ stdenvNoCC.mkDerivation rec {
   meta = with lib; {
     description = "A GeoLite2 data created by MaxMind";
     homepage = "https://github.com/Dreamacro/maxmind-geoip";
-    license = licenses.unfree;
+    license = licenses.cc-by-sa-40;
```

Comment:
I don't think it actually is cc-by-sa-40, but a custom (in many senses, nonfree) license, https://www.maxmind.com/en/geolite2/eula

Reply:
IANAL, this is taken from the v2ray-geoip which uses the same data source. In the 4 policies in the EULA:

[Creative Commons Corporation Attribution-ShareAlike 4.0 International License (the "Creative Commons License")](https://creativecommons.org/licenses/by-sa/4.0/)
    [MaxMind Data Processing Addendum ("DPA")](https://www.maxmind.com/data-processing-addendum.pdf)
    [MaxMind Privacy Policy ("PP")](https://www.maxmind.com/en/privacy-policy)
    [MaxMind Website Terms of Use ("WT")](https://www.maxmind.com/en/terms-of-use)

I thought only the CC license applies to the data and the others applies to the website. Alternatively, we can use `unfreeRedistributable` here.

Reply:
Let's postpone this issue?